lib/components/narrative/line-itin/connected-itinerary-body.js

```diff
@@ -37,6 +37,11 @@ const noop = () => {}
 
 const ItineraryBodyContainer = styled.div`
   padding: 0px 10px;
+
+  /* TODO: Find a better way of doing this */
+  table {
+    font-size: 14px;
```

Comment:
I don't think this is as much of a hack as you think it is!

Reply:
Should probably add a class name to the table someday

Reply:
I think since this is styled components you don't need to!